custodia/server/__init__.py

```diff
@@ -100,7 +101,14 @@ def _create_plugin(parser, section, menu):
 
 
 def parse_config(args):
-    parser = ConfigParser(interpolation=ExtendedInterpolation())
+    defaults = {
+        'fqdn': socket.getfqdn(),
```

Comment:
Btw we may also want to change the parameter name we set from 'fqdn' to 'hostname' so that we do stay honest about what we are serving back, given that sometimes 'hostname' will turn out to be the short hostname and not the fqdn. In practices in most cases this may just be fine.